test/e2e_env/universal/universal_suite_test.go

```diff
@@ -57,6 +58,8 @@ var _ = SynchronizedBeforeSuite(
 var _ = Describe("User Auth", auth.UserAuth)
 var _ = Describe("DP Auth", auth.DpAuth, Ordered)
 var _ = Describe("HealthCheck panic threshold", healthcheck.HealthCheckPanicThreshold, Ordered)
+var _ = Describe("HealthCheck", healthcheck.Policy, Ordered)
```

Comment:
I _think_ that if you have ordered here parallelized. You've got 2 Describe inside `healthcheck.Policy`. Place `Ordered` in each one of them, not here